src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
I *think* I see the discrepancy in understanding.

As I understand it, command files are a `gcc` specific feature. They're not a feature of the shell. Shell command files (e.g. `.bat`, `.sh`) will *also* have the character limit.

That doesn't bar special-casing `spawn()` to dissect the command, identify if the head of the command is a string corresponding to a utility that accepts the command file argument, and then proceeding from there.